base/src/jsMain/kotlin/runBlocking.kt

```diff
@@ -30,10 +32,9 @@ import kotlin.coroutines.CoroutineContext
  * @param block the coroutine code.
  */
 actual fun <T> runBlocking(context: CoroutineContext, block: suspend CoroutineScope.() -> T): T {
-    // this does not wait for the result unfortunately
     val result = GlobalScope.async { block(this) }
     while (!result.isCompleted) {
-        // DO NOTHING
+        GlobalScope.launch { delay(10) }
```

Comment:
Let's for now just go back to doing the async and nothing else.  

`runBlocking` is available in a mixed iOS/Android/JVM sourceset (the 'conurrent` sourceset). Because almost all sourcesets used to have JS it was not there. Now that is removed from many modules. So I propose in a follow up PR we deprecate kaluga `runBlocking`.

It's almost only used for tests which can use `runTest` instead if there is JS (though they should even if there is not probably). Probably in 90% of cases it's a drop in replacement.

Places where its actually used in kaluga outside of tests should be replacable (e.g. by one of the `runOnMain` style methods which could be noop on JS).